components/ftx/browser/buildflags/BUILD.gn

```diff
@@ -0,0 +1,9 @@
+import("//build/buildflag_header.gni")
+import("//brave/components/ftx/browser/buildflags/buildflags.gni")
+
+buildflag_header("buildflags") {
+  header = "buildflags.h"
+  flags = [
+    "FTX_ENABLED=$ftx_enabled",
```

Comment:
the convention is `enable_feature`

Reply:
fixed via https://github.com/brave/brave-core/compare/6f4c503619e82a440ef7db9782288e922c37d5e4..f4d887c0baf39543331107f2e22f9b0ad4986c02